src/screens/Learn/Glossary/Glossary.tsx

```diff
@@ -72,7 +77,8 @@ const Glossary: React.FC = () => {
         <LearnHeading1>{header}</LearnHeading1>
         {intro && <MarkdownContent source={intro} />}
         <LastUpdatedDate>
-          Last updated {formatNumericalDate(lastUpdatedDate)}
+          Last updated{' '}
+          {formatDateTime(parseDateString(lastUpdatedDate), timeFormats.L)}
```

Comment:
I feel like we might want to create a function to format a date string directly (we can do it later)

```ts
formatDateString(lastUpdatedDate, TimeFormat.L)
```